COMMPLAN.md

```diff
@@ -0,0 +1,204 @@
+Python-related discussion that doesn't fall into the more specific
+public streams (e.g., `jupyter`, `dask`).  NCAR-wide *announcements*
+should be made to the `announce` Public Stream.  In general, the name
+of the Public Stream defines the *subject* of its discussions.
+
+Private Streams (i.e., those labeled with a lock symbol) should be used
+for *internal* communications between people who have been invited to
+the Private Stream.  For example, the `Xdev` Private Stream is meant
+for internal communication within the Xdev Team.  In general, the
+name of the Private Stream defines the *group of people* included in
+the discussions.
+
+Zulip is also excellent for instant messaging *in general*, so if you
+are going to be watching Zulip to help out other Python users here at
+NCAR, then it makes sense to use it for direct communication amongst
+Xdev Team members, too.
+
+#### What should we *not* use it for?
+
+Don't use Zulip for communication that is meant for the
+*entire user community* (e.g., all of Pangeo), and not just NCAR staff.
+In those cases, I recommend using the Pangeo communication channels, as
+described in the [PANGEO Guide](PANGEO.md).
+
+### The Xdev Blog
+
+#### Why should we use it?
+
+Our blog (https://ncar.github.io/xdev) is an excellent way for us to
+communicate complex ideas, or to make announcements, to the *broader*
+community (i.e., beyond NCAR) in an easy-to-access form.  Blogs persist
+and are designed to be referenceable at a later date, which makes them
+excellent for sharing our individual (or Team) accomplishments in a
+way that can be cited.  By hosting our *own* blog, which is connected
+to Google Analytics, we can gather metrics and visit statistics on
+our blog posts, which can provide us useful information about who is
+making use of our posts.  Career-wise, blog posts are strongly
+encouraged in the software engineering community!
+
+#### What should we use it for?
+
+Use our blog as a way of publicizing new software package, new software
+versions, solutions to hard problems, or any general accomplishment.
+Use our blog to make *general public announcements* about things like
+tutorials or new services.  Use our blog to publicize Xdev Team work and
+achievements.
+
+#### What should we *not* use it for?
+
+Don't use the blog for rants or making complaints about anything.  The
+blog should only be used for *constructive* communication.
+
+### Twitter (@NCARXdev)
+
+#### Why we should use it?
+
+Twitter is a commonly used platform by many people in the Tech Industry,
+especially software engineers and scientists.  For example, a very large
+fraction of the Pangeo community regularly tweet.  Hence, Twitter makes
+an excellent platform for getting the attention of the general public
+(or the fraction of the general public interested in NCAR's Python
+work).
+
+#### What should we use it for?
+
+The Xdev Twitter account should be for making *announcements* to the
```

Comment:
I think we should make "The Xdev Twitter account" a link to [@NCARXdev](https://twitter.com/i/lists/1053067173961326594) here as well (or perhaps instead of in the subsection header?)

Reply:
Done